### ISSUE TITLE
Accessibility: unlabeled buttons

### ISSUE THREAD

**Issue by vvillenave** [johnfactotum/foliate#762]
Whilst Foliate’s interface is clean and fairly easy to navigate for  
visually-impaired users (minus the actual text-rendering panel, see
#760), there are a couple of buttons that are not properly vocalized.

I haven’t conducted a full audit of the whole interface, but the most  
glaring lack in this department is (as of 2.6.3 anyway)… the text-to-speech  
button itself, which just appears as a “toggle button” without any other  
mention.

A useful task would be to open Foliate with either the GTK Inspector  
or (better yet) the Accerciser application, which focuses on what’s  
exposed to the At-Spi bridge, and add labels and descriptions wherever  
needed; additionally (though not user-exposed but _so_ much more  
convenient for debugging and bug reporting) I’d suggest using GTK “id”  
attribute for all major areas, dialogs, widgets and UI elements.
